src/inmanta/logging.py

```diff
@@ -788,6 +791,27 @@ def apply_options(self, options: Options, component: str | None = None, context:
         Used to select which config file option to use (logging.component)
         :param context: context variables to use if the config file is a template
         """
+
+        def user_defined_options() -> Optional[str]:
+            """
+            Returns a string with the options (excluding "--verbose" and "--logging-config") that were set by the user
+            """
+            ignore_keys = ["verbose", "logging_config"]
+
+            option_to_cli = {
+                "log_file": "--log-file",
+                "log_file_level": "--log-file-level",
+                "timed": "--timed-logs",
+                "keep_logger_names": "--keep-logger-names",
+            }
```

Comment:
@arnaudsjs I couldn't find a way to figure out the cli name of the options from the Namespace option so I figured I would just do this. What do you think? And should I move this to const since I use it on the test or do you think this is fine?

Reply:
Not a big fan of this, but it does work